src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -302,10 +283,9 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     fontSize: normalizeFontSize(24)
   },
-  backbutton: {
-    paddingRight: 16,
-    justifyContent: 'center',
-    paddingLeft: dimen.normalizeDimen(20)
+  backButton: {
+    paddingRight: dimen.normalizeDimen(20),
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've removed paddingLeft from backButton style. If this change is intentional and doesn't affect the UI negatively, then it's fine. Otherwise, consider adding it back.



<!-- This is an auto-generated comment by OSS CodeRabbit -->